skycatalogs/catalog_creator.py

```diff
@@ -374,6 +376,10 @@ def __init__(self, parts, area_partition=None, skycatalog_root=None,
         self._sso_truth = self._sso_creator.sso_truth
         self._sso_sed = self._sso_creator.sso_sed
         self._sso_partition = sso_partition
+        self._run_options = run_options
+
+        # Do we need this?
+        self._sed_bins = None
```

Comment:
If this attribute were named something like `self._cosmodc2_sed_bins` or `self._tophat_sed_bins`, it would clearer in which contexts it's used.  Based on the code below, setting to `None` here isn't strictly needed, but setting a private attribute in the `__init__` seems like a good practice.

Reply:
Agreed.  I'll change the name.